backend/migrations/versions/d65d4263d860_add_date_assigned_to_usergoal_primary_.py

```
@@ -0,0 +1,67 @@
+    op.execute(
+        "INSERT INTO user_goals_new (user_id, goal_id, date_assigned, date_completed) SELECT user_id, goal_id, date_assigned, date_completed FROM user_goals"
+    )
+
+    # Drop the old table
+    op.drop_table("user_goals")
+
+    # Rename the new table to the old table's name
+    op.rename_table("user_goals_new", "user_goals")
+
+
+def downgrade():
```

Ruff (0.8.0)

Missing return type annotation for public function `downgrade`

Add return type annotation: `None`

(ANN201)